arch/arm/core/aarch32/cpu_idle.S

```diff
@@ -64,6 +68,11 @@ SECTION_FUNC(TEXT, z_arm_cpu_idle_init)
 	dsb
 	\wait_instruction
 
+#if defined(CONFIG_ARM_ON_EXIT_CPU_IDLE_HOOK)
+	/* Inline the macro provided by SoC-specific code */
+	arm_on_exit_cpu_idle_hook
```

Comment:
If we are going with the macro this should be all caps. If we are using an hook small caps are fine.

Reply:
renamed to `SOC_ON_EXIT_CPU_IDLE` (no hook word, all caps)